### DISCUSSION TITLE
I think Colab restriction can be fixed

### DISCUSSION THREAD

**Discussion by Rasad0** [TheLastBen/fast-stable-diffusion#2191] [Ideas]
I think colab is terminating the code not because of gpu usage but for the hosting to another server. This is written on the FAQ. also the shivamshirno verson is still running without any restriction. so I would like to request the developers if they can do anything with this.

**Comment by TheLastBen** [comment]
Dreambooth colab notebook is working fine

**Reply by Rasad0** [reply]
I dont have colab pro so it keeps terminating the code.
